commitlint.config.ts

```diff
@@ -206,11 +206,13 @@ module.exports = {
                         if (bodyStr !== ''){
 
                             function paragraphHasValidEnding(paragraph: string): boolean {
+                                let lastWordInParagraph = paragraph.split(' ')[paragraph.split(' ').length - 1]
```

Comment:
please end typescript lines with semicolon; and don't call paragraph.split(' ') twice

Reply:
Done